### PR TITLE
Add alias trick on xdebug documentation part

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Related @Seldaek comment: https://github.com/composer/composer/commit/7a06615bf06c5a391ab2d360907012529508e695#commitcomment-14599053

### CODE REVIEW

File: doc/articles/troubleshooting.md [hunk 1]
```diff
@@ -146,6 +146,27 @@ locating the related `zend_extension` directive and prepending it with `;` (semi
 ;zend_extension = "/path/to/my/xdebug.so"
 ```
 
+If you disable this extension and still want it to be added on `php` cli command, you can deal with aliases:
```
Comment (staabm) [inline doc/articles/troubleshooting.md]: I would add that this "workaround" only works un *nix systems

Comment (Seldaek) [inline doc/articles/troubleshooting.md]: Works fine for me on windows using bash but sure :p

**Comment by Seldaek** [comment]
Thanks

File: doc/articles/troubleshooting.md (hunk 1 above)
Comment (staabm) [inline doc/articles/troubleshooting.md]: you need to provide your custom windows bash with composer then ;)

Comment (Seldaek) [inline doc/articles/troubleshooting.md]: It's an optional but recommended download at 
http://www.mingw.org/wiki/msys :p

**Pull request merged**